src/main/java/axiomq/com/starwars/controllers/CharacterController.java

```diff
@@ -1,10 +1,13 @@
 package axiomq.com.starwars.controllers;
 
 import axiomq.com.starwars.services.CharacterService;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
+@AllArgsConstructor
```

Comment:
I guess it's a good practice to have @RequestMapping on top of your Controller class.